pyproject.toml

```diff
@@ -149,3 +150,35 @@ disallow_untyped_calls = false
 module = "torch.*"
 follow_imports = "skip"
 follow_imports_for_stubs = true
+
+[tool.ruff]
+select = ["E", "F", "W", "B"]
+fixable = ["E", "F", "W", "B"]
+ignore = ["B027"]
+# ruff is less lenient than pylint and does not make any exceptions
+# (for docstrings, strings and comments in particular).
+line-length = 115
```

Comment:
Shouldn't this match the black setting (`88`)?

Reply:
That would be correct. I copy that config from the pylint config. That is how they configure `ruff` for the pylint repo. We could merge it with 88 and think of generally increasing it separately.

Reply:
Changed it to 88